huawei-wmi.c

```diff
@@ -429,19 +428,35 @@ static int huawei_wmi_micmute_led_set(struct led_classdev *led_cdev,
 	}
 }
 
+static int huawei_wmi_kbdlight_set_auto(int level);
+
+static int huawei_wmi_kbdlight_led_set(struct led_classdev *led_cdev,
+		enum led_brightness brightness)
+{
+	return huawei_wmi_kbdlight_set_auto(brightness);
+}
+
 static void huawei_wmi_leds_setup(struct device *dev)
 {
 	struct huawei_wmi *huawei = dev_get_drvdata(dev);
 
-	huawei->cdev.name = "platform::micmute";
```

Comment:
The reasoning behind the namespace was discussed a long time ago in the Linux mailing list. I don't recall why tho

Reply:
I suppose you mean https://lkml.org/lkml/2019/5/23/265, but that's discussing the driver itself, not related to the led specifically (which already was `platform::` in the diff).

Reply:
The problem is that this would break existing programs that rely on this name. Could we keep both files referring to the same LED?

Reply:
Programs should _not_ relate on these names imo. It's never been declared as a stable API to use. Moreover, I doubt any userspace software actually uses that led directly, as opposed to the native kernel trigger used by ALSA (which obviously is still there).